python/src/equistore/operations/reduce_over_samples.py

```diff
@@ -245,3 +303,60 @@ def mean_over_samples(tensor: TensorMap, samples_names: List[str]) -> TensorMap:
     return _reduce_over_samples(
         tensor=tensor, samples_names=samples_names, reduction="mean"
     )
+
+
+def std_over_samples(tensor: TensorMap, samples_names: List[str]) -> TensorMap:
+    r"""Create a new :py:class:`TensorMap` with the same keys as
+    as the input ``tensor``, and each :py:class:`TensorBlock` is obtained
+    performing the std deviation of the corresponding input :py:class:`TensorBlock`
+    over the ``samples_names`` indices.
```

Comment:
We can leave this for now, but should try to shorten the "short" description into a line.